kolibri/core/assets/src/views/tabs/index.vue

```diff
@@ -1,76 +1,33 @@
 <template>
 
-  <ui-tabs
-    ref="tabs"
-    class="tabs"
-    :type="type"
-    :disableRipple="true">
-    <ui-tab
-      v-for="item in items"
-      :title="item.title"
-      :icon="item.icon"
-      :selected="itemIsSelected(item)"
-      :disabled="itemIsDisabled(item)"/>
-  </ui-tabs>
+  <nav class="tabs">
+    <ul>
+      <slot/>
```

Comment:
@DXCanas would you mind posting a pseudo-code example of how scoped slots could be used for this scenario?

When @christianmemije and I discussed, the main goal was to avoid defining the list of tabs in terms of a javascript array of config objects.